dev_docs/tutorials/building_a_kibana_distributable.mdx

```diff
@@ -0,0 +1,135 @@
+---
+id: kibDevTutorialBuildingDistributable
+slug: /kibana-dev-docs/tutorial/building-distributable
+title: Building a Kibana distributable
+summary: Learn how to build a Kibana distributable
+date: 2021-05-10
+tags: ['kibana', 'onboarding', 'dev', 'tutorials', 'build', 'distributable']
+---
+
+On the course of that tutorial it will be possible to learn more about how we can create a Kibana distributable 
+as well as handle its different main configurations.
+
+At any given point, you can get CLI help running the following command:
+
+```bash
+yarn build --help
+```
+
+## Prerequisites
+
+For the basic steps of the build we only require you to use one of the following operating systems:
+
+- Linux
+- macOS
+
+However if as part of the build you also want to generate Linux installation packages (deb, rpm) or Docker images there 
+are other dependencies to have in mind as those installation packages and docker images generation are run using fpm, dpkg, rpm, and Docker.
+
+### For Linux Installation Packages Build
+
+Please make sure you have the following dependencies installed:
+
+#### Install FPM dependencies
+
+**On OSX/macOS:**
+
+```bash
+brew install gnu-tar rpm
+```
+
+**On Red Hat systems (Fedora 22 or older, CentOS, etc):**
+
+```bash
+yum install ruby-devel gcc make rpm-build rubygems
+```
+
+**On Fedora 23 or newer:**
+
+```bash
+dnf install ruby-devel gcc make rpm-build libffi-devel
+```
+
+**On Oracle Linux 7.x systems:**
+
+```bash
+yum-config-manager --enable ol7_optional_latest
+yum install ruby-devel gcc make rpm-build rubygems
+```
+
+**On Debian-derived systems (Debian, Ubuntu, etc):**
+
+```bash
+apt-get install ruby ruby-dev rubygems build-essential
+```
+
+#### Install FPM
+
+Finally install fpm with the gem tool with:
+
+```bash
+gem install fpm -v 1.5.0
+```
+
+### For Docker Images Build
+
+For Docker, the installation instructions can be found at [Install Docker Engine](https://docs.docker.com/engine/install/).
```

Comment:
thoughts on consistent casing for `Docker`?  there's a few instances of lower case below

Reply:
👍 Done

Reply:
@jbudz do you think we are good here now?